src/ReverseProxy/LoadBalancing/SlowStartPowerOfTwoChoicesLoadBalancingPolicy.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Yarp.ReverseProxy.Model;
+using Yarp.ReverseProxy.Utilities;
+
+namespace Yarp.ReverseProxy.LoadBalancing;
+
+internal sealed class SlowStartPowerOfTwoChoicesLoadBalancingPolicy : ILoadBalancingPolicy
```

Comment:
>rtPowerOfTwoChoices [](http://example.com/codeflow?start=29&length=19)

PowerOfTwoChoices is such a confusing name.
Are you having fun?
Can you call it SlowLeastBusyLoadBalancingPolicy?
If you mind that having just two candidates is not adequately represented in the name, you can generalize the policy to choose out of N, and set N = 2, subject to a adjustment.